lhotse/dataset/feature_transforms.py

```diff
@@ -1,15 +1,30 @@
 import torch
 
+from typing import Union
+
 from lhotse import CutSet
+from lhotse.utils import Pathlike
+
 
+class GlobalMVN(torch.nn.Module):
+    """Apply global mean and variance normalization"""
 
-class Standardize:
-    def __init__(self, cuts: CutSet):
+    def __init__(self, norm_means, norm_stds):
```

Comment:
I think it's more "pytorch-y" if you give it a dimension arg (`feature_dim: int`); then it should enable using `load_state_dict` (and you can adjust `from_cuts` to use that too);

Can you add `to_file()` too? That will make it convenient to dump and load later.

Reply:
And initialize the buffers to zeros and ones? So something like

```
def __init__(self, feature_dim: int):
    self.register_buffer("norm_means", torch.zeros(feature_dim))
    ...

@classmethod
def from_cuts(cls, cuts):
    # compute stats
    global_mvn = cls(feature_dim_that_we_retrieved_from_cuts)
    global_mvn.norm_means = stats["norm_means"]
    ...
    return global_mvn
```
Yup will add `to_file` too

Reply:
Looks good to me

Reply:
Please dont merge yet, i will add one more test tomorrow.

Reply:
Should be ready now.